Makefile

```diff
@@ -59,7 +59,7 @@ endif
 
 
 ifeq ($(BUILD_OMS),BUILD_OMS)
-dsckit098: nx nxOMSAgent nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof
+dsckit098: nx nxOMSAgent nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof nxOMSAutomationWorker
```

Comment:
@sourabhguha @D1v38om83r @robbiezhang This worker would be used in non-OMS scenarios as well, like DIY. Would DSC be used to send the bits to the machines in the non-OMS scenario? Just asking to confirm the nxOMS- prefix.